content/en/monitors/service_level_objectives/_index.md

```diff
@@ -110,6 +110,37 @@ Hover over a saved view from the list and select the hyperlink icon to copy the
 
 Once you are using a saved view, you can update it by selecting that saved view, modifying the query, and clicking the *Update* button below its name in the *Saved Views* panel. To change the saved view's name or delete a saved view, hover over its row in the *Saved Views* panel and click the pencil icon or trash can icon, respectively.
 
+## SLO Audit Events
+
+SLO audit events allow you to track the history of your SLOs' configurations using the Event Stream. Audit events will be added to the Event Stream every time you create, modify or delete an SLO. Each event will include information on an SLO's configuration and combined together give you a history of how the SLO's configuration changes over time. 
+
+The following SLO configuration information will be included:
+
+- Name
+- Description 
+- Target percentages and time windows
+- Datasources (monitor IDs or metric query)
+
+Three different SLO audit events will appear in the Event Stream:
+
+1. `SLO Created` events will show all four pieces of configuration information at creation time of the SLO
+2. `SLO Modified` events will show a diff of only the aspects of the configuration that were changed during a modification
+3. `SLO Deleted` events will show all four pieces of configuration information the SLO had right before it was deleted
+
+To get a full list of all SLO audit events, enter the search query `tags:audit,slo` in the Event Stream. To view the list of audit events for a specific SLO, instead enter `tags:audit,slo_id:<SLO ID>` with the ID of the desired SLO.
+
+You can also query the Event Stream programmatically using the [Datadog Events API][8].
+
+**Note:** If you don't see events appear in the UI, be sure to set the time frame of the Event Stream to a longer period like the past 7 days.
+
+{{< img src="monitors/service_level_objectives/slo-audit-events.png" alt="SLO audit events"  >}}
```

Comment:
We should definitely blur your name and email address, and we usually blur profile pics too. Let me know if you need a hand doing that (CloudApp is how I usually do it).